src/visualizations/App.fsproj

```
@@ -5,7 +5,6 @@
     <ItemGroup>
         <Compile Include="Types.fs" />
         <Compile Include="Highcharts.fs" />
-        <Compile Include="GoogleCharts.fs" />
         <Compile Include="Utils.fs" />
```

For those using visual studio add it to the project file:
```suggestion
        <Compile Include="GoogleCharts.fs" />
        <Compile Include="Utils.fs" />
```